.github/workflows/run-harvest-check.yml

```diff
@@ -7,7 +7,7 @@ on:
   workflow_dispatch:
 
 jobs:
-#   # TODO: delete when reenabled
+#   # Note: run this action if the harvesters need to be commented out
```

Comment:
Why would the harvesters need to be commented out?

Reply:
If we ever want to "turn off the harvesters" again just like we needed to for the transition.  I'm not foreseeing anything specifically, just mentioning.